src/job/job.spec.js

```diff
@@ -94,4 +94,19 @@ describe("job", () => {
       ]).toMatchSnapshot();
     });
   });
+
+  describe(".get() :", () => {
+    it("gets an existing job", async () => {
+      const id = await xo.createJob(defaultJob);
+      const job = await xo.call("job.get", { id });
+      expect(typeof job).toBe("object");
```

Comment:
No need to test the type of the result. If it change, the snapshot will not correspond with it.